### PR TITLE
release 4.18.0

### PR DESCRIPTION
**Pull request by Blackbaud-ToddRoberts**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.18.0](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.18.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.18.0 (2021-07-19)
+
+- Added default theme sytles for box and elevation classes. [#246](https://github.com/blackbaud/skyux-theme/pull/246)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Added default theme styles for box and elevation classes. [#246](https://github.com/blackbaud/skyux-theme/pull/246)
```

Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md:3]: typo "sytles"

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.18.0](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-4.18.0)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**